Tugas Kelompok - 4/src/Main.java

```diff
@@ -1,9 +1,55 @@
+import java.util.Scanner;
 public class Main {
+        static int[] data = new int[5];
+        static Scanner input = new Scanner(System.in);
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        int choice;
+        do {
+            System.out.println(""
+                        + "\nSelamat Datang di Program Simulasi"+"\n"
+                        + "Menu"+"\n"
+                        + "1. Random Data"+"\n"
+                        + "2. Simulasi Bubble Sort - Ascending"+"\n"
+                        + "3. Simulasi Selection Sort - Ascending"+"\n"
+                        + "4. Simulasi Bubble Sort - Descending"+"\n"
+                        + "5. Simulasi Selection Sort - Descending"+"\n"
+                        + "6. Keluar"+"\n"
+                        + "Masukkan Pilihan Anda."+"\n");
+                        
+        choice = input.nextInt();
+ 
         int[] sampleArray = { 5, 1, 4, 2, 8 };
+        
+        switch (choice) {
+            case 1:
+                randomData();
+                break;
+            case 2:
+                bubbleSortAscending(sampleArray);
```

Comment:
Nah bagian ini bisa dibikin 

```
int[] arrayData = this.data[0] ? this.data || sampleArray;

// Kemudian implementasinya
bubbleSortAscending(arrayData);
```

Maksudnya kalo `this.data` ada isi datanya, maka pake data, kalo enggak ada datanya maka pakai default array